### DISCUSSION TITLE
Notifications disabled in InTune, yet aren't actually disabled.

### DISCUSSION THREAD

**Discussion by WalterSobchak9515** [Weatherlights/Winget-AutoUpdate-Intune#46] [General]
As stated in the title, notifications are disabled yet they still show every notification on each endpoint:

![image](https://github.com/user-attachments/assets/d1091ebf-4654-4397-8477-38977ab8d3d7)

![image](https://github.com/user-attachments/assets/27f5c647-63c7-4ec8-8700-cb581d965503)

![image](https://github.com/user-attachments/assets/bcba216f-6875-47ca-81e8-fdc9825ebe71)

![image](https://github.com/user-attachments/assets/6ceb0ba5-cf79-4002-bb51-8b010a08ddf8)

How can I get the notifications to disable?

**Comment by ak47uk** [comment]
Have you tried enabling the policy then picking a setting from the dropdown? I can’t remember what options there are but there might be one for no notifications.

**Comment by WalterSobchak9515** [comment]
Sorry, I should have included a screenshot showing that as well.  The notifications have been set to disabled as well as the drop down menu has been set to none for well over a month, yet every single notification is still shown on all PC's.  Here you go:

![image](https://github.com/user-attachments/assets/78e64bef-dce5-4cee-8479-cb7c7846944e)

**Reply by ak47uk** [reply]
You need to set it to Enabled for that policy to apply, then select None from the dropdown. Can you do that and test?

**Reply by WalterSobchak9515** [reply]
Yes, of course, with it set the way it is now of course I am getting notifications.  I am out of town for the next 8 or 9 days so I will change the policy but won't be able to thoroughly test until I get back in town.

Thanks for the help.
